normative_rule_defs/zihintntl.yaml

```diff
@@ -8,6 +8,7 @@ $schema: "../docs-resources/schemas/defs-schema.json#"
 
 chapter_name: Zihintntl (Non-tmp Hint)
 
+
```

Comment:
Should this be a normative rule?

`NTL instructions must not void the forward-progress guarantee provided by the LR/SC instructions`